android/src/main/java/com/launchdarkly/reactnative/LaunchdarklyReactNativeClientModule.java

```diff
@@ -774,8 +774,13 @@ public void registerAllFlagsListener(final String listenerId, final String envir
         LDAllFlagsListener listener = new LDAllFlagsListener() {
             @Override
             public void onChange(List<String> flagKeys) {
+                WritableArray flagKeysArray = Arguments.createArray();
+                for (String flagKey: flagKeys) {
+                    flagKeysArray.pushString((flagKey));
+                }
```

Comment:
I wasnt sure about using the for each loop or using 

```java
flagKeys.forEach(flagKeysArray:: pushString)
```

the above gave me a warning in android studio about sdk versions so I decided going to basic option would be most compatible.